frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/PipelineWorkspacesSection.tsx

```diff
@@ -108,6 +108,7 @@ const PipelineWorkspacesSection: React.FC = () => {
                 }
                 fullWidth
                 required={!workspace.optional}
+                helpText={workspace.description}
```

Comment:
Translation is missing here.

Reply:
Dynamic Translations are not a thing we are currently are supporting. No way to take user-data and translate it on the fly without a 3rd party system which we do not have.